server/src/uuid/model/entity/mod.rs

```diff
@@ -918,19 +917,10 @@ impl Entity {
     where
         E: Executor<'a>,
     {
-        let after_db_time = payload
-            .after
-            .as_ref()
-            .map(|after| {
-                chrono::DateTime::parse_from_rfc3339(after).map_err(|_| {
-                    operation::Error::BadRequest {
-                        reason: "The date format should be YYYY-MM-DDThh:mm:ss{Timezone}"
-                            .to_string(),
-                    }
-                })
-            })
-            .transpose()?
-            .map(|date| date.with_timezone(&Berlin).to_string());
+        let after_db_time = match payload.after.as_ref() {
+            Some(date) => DateTime::parse_from_rfc3339(date)?,
+            None => DateTime::now(),
+        };
```

Comment:
Can be a helper function

Reply:
true, let's go on for now and we could do this refactoring if the need is bigger